### PR TITLE
Modules, wire into Guice

### LINKED ISSUE #81: Actually make stuff work
Wire the modules. c'mon.

### PR DESCRIPTION
**Pull request by cli1903**

### CODE REVIEW

File: frontend/ui/lib/widgets/create_trip_widget.dart [hunk 1]
```diff
@@ -66,8 +66,7 @@ class _CreateTripWidgetState extends State<CreateTripWidget> {
             padding: const EdgeInsets.all(25.0),
             child: MapWidget(),
           ),
-          Container(
-              child: Text('$newInformation')),
+          Container(child: Text('$newInformation')),
```
Comment (KoopaKing) [inline frontend/ui/lib/widgets/create_trip_widget.dart]: please remove this from the review.

File: frontend/src/main/java/com/google/tripmeout/frontend/servlet/ServletsModule.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.google.tripmeout.frontend.servlet;
+
+import com.google.inject.servlet.ServletModule;
+
+/**
+ * A {@link ServletModule} configured to serve SPS application paths.
```
Comment (KoopaKing) [inline frontend/src/main/java/com/google/tripmeout/frontend/servlet/ServletsModule.java]: Can you remove the "SPS"?

File: frontend/src/main/java/com/google/tripmeout/frontend/storage/StorageBindingModule.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.google.tripmeout.frontend.storage;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.storage.InMemoryPlaceVisitStorage;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+
+public class StorageBindingModule extends AbstractModule {
```
Comment (KoopaKing) [inline frontend/src/main/java/com/google/tripmeout/frontend/storage/StorageBindingModule.java]: Can we call this the InMemory_ if it only provides in memory stuff? Ideally at some point we will an `InMemoryStorageModule` and a `DatastoreStorageModule` that can be wired depending on what we want to do (like local testing vs deploying).

File: frontend/src/main/java/com/google/tripmeout/frontend/storage/StorageBindingModule.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  @Override
+  public void configure() {
+    bind(PlaceVisitStorage.class).to(InMemoryPlaceVisitStorage.class).in(Singleton.class);
+    bind(TripStorage.class).to(InMemoryTripModelStorage.class).in(Singleton.class);
+  }
+}
```
Comment (chuckluv) [inline frontend/src/main/java/com/google/tripmeout/frontend/storage/StorageBindingModule.java]: nit: new line at  end of file

**Review by bsidhom — APPROVED** [review]
LGTM

**Review by chuckluv — APPROVED** [review]

**Pull request merged**